cmd/attest.go

```diff
@@ -69,12 +76,23 @@ hardware and guarantees a fresh quote.
 
 		attestOpts := client.AttestOpts{}
 		attestOpts.Nonce = nonce
-		if len(teeNonce) != 0 {
-			attestOpts.TEENonce = teeNonce
-			attestOpts.TEEDevice, err = client.CreateSevSnpDevice()
-			if err != nil {
-				return fmt.Errorf("failed to collect TEE attestation report: %v", err)
+		if len(teeTechnology) != 0 {
+			// Add logic to open other hardware devices when required.
+			switch teeTechnology {
```

Comment:
I feel like it is a bit confusing with a switch block inside an if block.
Can you just use the switch block and add the logic under `else if len(teeNonce) != 0 {` as part of the `case ""`?

Reply:
As we discussed , I have used a `switch case` instead of `if else` and it is working fine with all possible cases.